linkerd/app/src/lib.rs

```diff
@@ -147,29 +153,54 @@ impl Config {
             let span = info_span!("outbound");
             let _enter = span.enter();
             info!(listen.addr = %outbound_addr);
-            tokio::spawn(
-                serve::serve(
-                    outbound_listen,
-                    outbound::server::stack(
-                        &outbound,
-                        dst.profiles.clone(),
-                        dst.resolve,
-                        outbound::tcp::connect::stack(
-                            &outbound.proxy.connect,
-                            outbound_addr.port(),
-                            local_identity.clone(),
+            if ingress_mode {
```

Comment:
nit: maybe worth an `info` log here indicating whether or not we're in ingress mode?